src/amo/components/SearchFilters/styles.scss

```diff
@@ -18,3 +18,85 @@
 #SearchFilters-Recommended {
   @include margin-end(10px);
 }
+
+.SearchFilters-ThemeColors {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+
+  .SearchFilter-ThemeColors-ColorItem {
+    cursor: pointer;
+    outline: 2px solid transparent; /* originally had a 1px border instead, but it was too subtle */
+    border: 1px solid $grey-40;
+    margin-right: 4px;
+    width: 34px;
+    height: 34px;
+    display: inline-block;
+    border-radius: 50%;
+    vertical-align: top;
+
+    &.SearchFilter-ThemeColors-ColorItem--active {
+      outline-color: $action-active-color;
+    }
+
+    input[type='color'] {
+      width: 100%;
+      height: 100%;
+      opacity: 0; /* hidden, but captures clicks to trigger the color dialog */
+      padding: 0;
+      border-radius: 50%;
+      border: 0;
+    }
+  }
+}
+
+.SearchFilter-ThemeColors-ColorItem--custom {
+  background: linear-gradient(
+    90deg,
+    hsl(0deg, 100%, 50%),
```

Comment:
Would be kind of cool easter egg if this were a gradient of mozilla branded colors. Nice to have but I think it would be cool.

Reply:
I'll take suggestions of what the `linear-gradient` exact rule should be :sweat_smile: I struggled with coming up with something that looks good enough